spotify/track.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Why there's a new file? If it's a file for a new API please move it to a new PR.